packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -395,6 +394,9 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         status_ = claimData[0].counteredBy == address(0) ? GameStatus.DEFENDER_WINS : GameStatus.CHALLENGER_WINS;
         resolvedAt = Timestamp.wrap(uint64(block.timestamp));
 
+        // Try to update the anchor state, but do not revert if it fails.
+        try ANCHOR_STATE_REGISTRY.tryUpdateAnchorState() { } catch { }
+
         emit Resolved(status = status_);
```

Comment:
This must be moved above the `tryUpdateAnchorState` call. There's a check for the status in that function, and the status must be updated before it's called.